gallery/src/components/NxDateInput/NxDateInputPage.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode, NxP, NxTextLink }
+  from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxDateInputSimpleExample from './NxDateInputSimpleExample';
+
+const NxDateInputSimpleExampleCode = require('./NxDateInputSimpleExample?raw');
+
+const NxTextInputPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        Standard HTML date input with validation styling.
+        It is using <NxTextLink href="#/pages/NxTextInput">NxTextInput</NxTextLink>
+        under the hood and has similar features and properties.
```

Comment:
Missing whitespace in the render before "under"

Reply:
> Missing whitespace in the render before "under"

Fixed: 815e112b791a0668bad7d24acd0349394d891017